### PR TITLE
- Update block labes for consistency.

### PR DESCRIPTION
**Pull request by divyarajmasani**
## Context

Some of the labels were not following proper names.

## Summary

This PR updates labels used for the Web Stories settings.

### CODE REVIEW

File: assets/src/web-stories-block/block/index.js [hunk 1]
```diff
@@ -33,8 +33,8 @@ import transforms from './transforms';
 const { name, category, attributes } = metadata;
 
 const settings = {
-  title: __('Web Stories List', 'web-stories'),
-  description: __('Embed Web Stories.', 'web-stories'),
+  title: __('Web Stories', 'web-stories'),
+  description: __('Embed web stories.', 'web-stories'),
```
Comment (spacedmonkey) [inline assets/src/web-stories-block/block/index.js:37]: Let's leave this as `Web Stories`

Comment (divyarajmasani) [reply assets/src/web-stories-block/block/index.js]: I've revert the changes.

File: assets/src/web-stories-block/block/components/storiesInspectorControls.js [hunk 2]
```diff
@@ -170,7 +170,7 @@ const StoriesInspectorControls = (props) => {
   const previewLink = select('core/editor').getEditedPostPreviewLink();
   const carouselMessage = createInterpolateElement(
     __(
-      `<b>Note:</b> Carousel view's functionality will not work in Editor. <a>Preview</a> post to see it in action.`,
+      `<b>Note:</b> Carousel functionality will not work in Editor. <a>Preview</a> post to see it in action.`,
       'web-stories'
     ),
```
Comment (swissspidy) [inline assets/src/web-stories-block/block/components/storiesInspectorControls.js:175]: Have we ever gotten Glider.js to work in the editor?

Comment (divyarajmasani) [reply assets/src/web-stories-block/block/components/storiesInspectorControls.js:175]: I believe we had kept it on good to have priority, I was not able to work on it. But if it's something we must include then I may create a separate PR for that.

Comment (divyarajmasani) [reply assets/src/web-stories-block/block/components/storiesInspectorControls.js:175]: Certainly, I'll create a new issue with the information I've.

Comment (divyarajmasani) [reply assets/src/web-stories-block/block/components/storiesInspectorControls.js:175]: @swissspidy draft PR for the carousel in editor. I found some changes in my local branches/stash and was able to build on that. Check the screen grab added to see the expected diff.

https://github.com/rtCamp/web-stories-wp/pull/322

**Comment by divyarajmasani** [comment]
If this PR looks good, shall we go ahead and merge?

**Review by swissspidy — APPROVED** [review]

**Pull request merged**